qadence/ml_tools/data.py

```diff
@@ -82,34 +82,34 @@ def to_dataloader(*tensors: Tensor, batch_size: int = 1, infinite: bool = False)
 
 
 @singledispatch
-def data_to_device(xs: Any, device: torch_device) -> Any:
+def data_to_device(xs: Any, *args: Any, **kwargs: Any) -> Any:
     """Utility method to move arbitrary data to 'device'."""
-    raise ValueError(f"Cannot move {type(xs)} to a pytorch device.")
+    raise ValueError(f"Unable to move {type(xs)} to {args}, {kwargs}.")
```

Comment:
It feels like since `Any` type is passed, this message could potentially be non-sensical.

Reply:
but it will check the _actual_ type of `xs` and log that. so str, int etc

Reply:
good idea